docs/integrations/sources/wordpress.md

```diff
@@ -0,0 +1,10 @@
+# Wordpress
+
+[Wordpress](https://wordpress.org/) is the most popular tool for creating websites worldwide.
+
+## Sync overview
+
```

Comment:
I know you can connect to the mysql db for self-hosted wordpress, but wasn't able to find consistent docs for connecting on hosted wordpress, since there are different providers, maybe throw in a 'ask your rep style'?